client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-pattern-pages.ts

```diff
@@ -1,11 +1,11 @@
 import { useSearchParams } from 'react-router-dom';
 import { INITIAL_PAGES, ORDERED_PATTERN_PAGES_CATEGORIES } from '../constants';
 import { useCategoriesOrder } from '../hooks';
-import { getPagePatternTitle } from '../utils';
+import { getPagePatternTitle, isPriorityPattern } from '../utils';
 import type { Pattern, Category, CustomPageTitle } from '../types';
 
 const usePatternPages = (
-	pagesMapByCategory: Record< string, Pattern[] >,
+	pageCategoryPatternsMap: Record< string, Pattern[] >,
```

Comment:
I renamed the prop to `pageCategoryPatternsMap` because that's the name of the real variable.